perfmetrics/scripts/fetch_metrics.py

```diff
@@ -51,15 +73,21 @@ def _parse_arguments(argv):
 
   args = _parse_arguments(argv)
 
-  if args.upload:
-    temp = fio_metrics_obj.get_metrics(args.fio_json_output_path, FIO_WORKSHEET_NAME)
-  else:
-    temp = fio_metrics_obj.get_metrics(args.fio_json_output_path)
+  bigquery_obj = bigquery.ExperimentsGCSFuseBQ(constants.PROJECT_ID, constants.DATASET_ID)
```

Comment:
We should only create objects when used. bigquery_obj will not be used if upload_bg is not passed.